internal/command/views/test.go

```diff
@@ -729,10 +729,7 @@ func (t *TestJSON) TFCStatusUpdate(status tfe.TestRunStatus, elapsed time.Durati
 }
 
 func (t *TestJSON) TFCRetryHook(attemptNum int, resp *http.Response) {
-	t.view.log.Error(
-		t.RetryLogHook(attemptNum, resp, false),
-		"type", json.MessageTestRetry,
-	)
+	t.Cloud.RetryLog(attemptNum, resp)
```

Comment:
You could get rid of the `TFCRetryHook` function entirely now, and just call the `RetryLog` function directly from wherever this was being called before.